### PR TITLE
refactored common step

### PR DESCRIPTION
**Pull request by AlexaVB**
Refactored step 'I am taken to step two' so that it uses two variables, the step number and the url.  This takes it down to one step instead of one for each page.   The new step checks the step number on the page and the url.

### CODE REVIEW

File: features/step_eight.feature [hunk 1]
```diff
@@ -59,7 +59,7 @@ Feature: Step eight page
   Scenario: No income
     Given I am a single person on step eight
     When I select no income
-    Then I am taken to step eleven
+    Then I am taken to step '11''probate'
```
Comment (leoapost) [inline features/step_eight.feature]: I think we can remove these single quotes
`Then I am taken to step 11 probate`

or at least the ones in the middle
`Then I am taken to step '11 probate'`

Comment (AlexaVB) [inline features/step_eight.feature]: Hey @leoapost, it needs the quotes to pick up the step variables:

Then(/^I am taken to step '(\d+)''([^\"]*)'$/) do |step, url|
expect(current_path).to eq '/questions/fee'		 
expect(common_page.content.p.text).to have_content step
expect(current_path).to eq '/questions/' + url
end

Comment (leoapost) [inline features/step_eight.feature]: how about to change the regex from `'(\d+)''([^"]*)'` to `'(\d+) ([^"]*)'`?

Comment (leoapost) [inline features/step_eight.feature]: `(\d+) ([^"]*)` should pick up the `Then I am taken to step 11 probate`
and
`'(\d+) ([^"]*)'` should pick up the `Then I am taken to step '11 probate'`

Any of the above is fine. Personally, I would go with the first one - less characters, easier to read.

Comment (AlexaVB) [inline features/step_eight.feature]: It does, you're right, I'll change it to the top one

**Pull request merged**